rites/roman1969/src/locales/pt-br.ts

```diff
@@ -220,6 +234,7 @@ export const locale: Locale = {
     bede_the_venerable_priest: 'São Beda Venerável, presbítero e doutor da Igreja',
     benedict_of_aniane_abbot: 'São Bento de Aniane, abade',
     benedict_of_nursia_abbot: 'São Bento, abade',
+    benedict_the_moor_religious: 'São Benedito, o Negro, religioso',
```

Comment:
Missing source. You could update the reference with a another source if you find a better one.

```suggestion
    benedict_the_moor_religious: 'São Benedito, o Negro, religioso', // src: https://www.scribd.com/document/697173100/Direto-rio-da-Liturgia-2024
```